src/StatusSummary.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+const Table = styled.table`
+  border-collapse: collapse;
+  text-align: center;
+  font-family: 'Gill Sans', sans-serif;
+`
+
+const CellHeader = styled.td`
+  color: NavajoWhite;
+  border: none;
+  font-size: 26px;
+  padding: 1px 10px 1px 10px;
+  letter-spacing: 1px;
+`
+
+const CellGreen = styled.td`
+  border: 3px solid #666666;
+  padding: 10px 10px 10px 10px;
+  font-size: 22px;
+  background: mediumseagreen;
+`
+
+const CellYellow = styled(CellGreen)`
+  background: goldenrod;
+`
+
+const CellRed = styled(CellGreen)`
+  background: indianred;
+`
+
+const StatusSummary = ({data}) => {
+  let vacant = 0
+  let ocupied = 0
+  let reserved = 0
+  data.map(d => {
+    if (d.status == 'vacant') {
+      vacant++
+    }
+    else if (d.status == 'ocupied') {
+      ocupied++
+    }
+    else if (d.status == 'reserved') {
+      reserved++
+    }
+  })
+  return(
+    <div>
+      <Table>
+        <thead>
+          <tr>
+            <CellHeader colSpan='2'>SUMMMARY</CellHeader>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <CellGreen>vacant</CellGreen><CellGreen>{vacant}</CellGreen>
+          </tr>
+          <tr>
+            <CellYellow>reserved</CellYellow><CellYellow>{reserved}</CellYellow>
+          </tr>
+          <tr>
+            <CellRed>ocupied</CellRed><CellRed>{ocupied}</CellRed>
+          </tr>
+        </tbody>
+      </Table>
+    </div>
+  )
+}
+
+StatusSummary.propTypes = {
+  data: PropTypes.array
```

Comment:
You can chain the proptype with `isRequired` to make sure a warning is shown if the prop isn't provided
-> `data: PropTypes.array.isRequired`